src/api/app/assets/stylesheets/webui/staging-workflow.scss

```diff
@@ -135,17 +135,17 @@ ul .table-list-group-item {
 
       &.state-ready {
         @extend .bg-success;
-        background-color: $request-ready-color;
+        background-color: $request-ready-color !important;
       }
 
       &.state-review {
         @extend .bg-warning;
-        background-color: $request-review-color;
+        background-color: $request-review-color !important;
```

Comment:
`!important` is effectively reverting what the previous line `@extend .bg-warning` does, so we can instead remove the previous line and avoid using `!important`.